spec/active_model/composite_command_spec.rb

```diff
@@ -110,5 +110,65 @@ def exception_command
         end
       end
     end
+
+    context "when composite command uses with transaction helper" do
```

Comment:
I added a new context but I think we just could reuse the previous context and use `with_transaction` [here](https://github.com/goldstar/active_model_command/blob/master/spec/examples/composite_command.rb#L12)